doc/openshift-guide.md

```diff
@@ -177,48 +177,54 @@ where `ec2-54-321-67-89.compute-1.amazonaws.com` is the Public Domain, and `54.3
  Ignore the suggested next steps in the text output at the command prompt and proceed to the next step below.
 
 3. Create a new Secret to reference your project by replacing *ACCESS_TOKEN* and MYDOMAIN with yours.
-  
+
  <pre><code>oc secret new-basicauth threescale-portal-endpoint-secret --password=https://ACCESS_TOKEN@MYDOMAIN-admin.3scale.net</code></pre>
 
  The response should look like this:
 
  <pre><code>secret/threescale-portal-endpoint-secret</code></pre>
 
-4. Create an application for your 3scale API Gateway from the template:
+4. Create an application for your APIcast Gateway from the template, and start the deployment:
 
  <pre><code>oc new-app -f https://raw.githubusercontent.com/3scale/apicast/v2/openshift/apicast-template.yml</code></pre>
 
  You should see a message indicating _deploymentconfig_ and _service_ have been successfully created.
 
-### Deploying 3scale API Gateway
+### Deploying APIcast Gateway
 
 1. Open the web console for your OpenShift cluster in your browser: https://OPENSHIFT-SERVER-IP:8443/console/
 
  You should see the login screen:
  <img src="https://support.3scale.net/images/screenshots/guides-openshift-login-screen.png" alt="OpenShift Login Screen">
 
-2. Login using your credentials created or obtained in the _Setup OpenShift_ section above.
+2. Login using your _developer_ credentials created or obtained in the _Setup OpenShift_ section above.
```

Comment:
Ah, actually it wasn't specified which user to use before... :) That's the problem.